episodes/chapter8.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Work on your own project"
+teaching: 10
+exercises: 1
+---
+
+:::::::::::::::::::::::::::::::::::::: questions
+
+- How can you use vantage6 in your own project?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Start working on your own research project
+
+OR
+
+- Complete some advanced challenges
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+# Working on your own project
+
+In this chapter, we will give you the opportunity to work on your own project. Feel free
+to ask questions and discuss your project with the instructors. We are here to help you!
+
+If you prefer, you can also complete some advanced challenges that we have prepared for
+you.
+
+# Advanced challenges
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Add privacy filters to your algorithm
+
+In the [previous lesson](./algorithm_development.md), you have created a simple
+algorithm. Now, you can add privacy filters to your algorithm.
+
+- Create an algorithm that does not return the results unless there are more than 10
+  data points.
+- Make this value configurable by the node administrator. Hint: look up the node
+  configuration options in the [documentation](https://docs.vantage6.ai) to provide
+  environment variables to your algorithm.
+- What else could you do to protect the privacy of the data?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Document your algorithm
+
+In the [previous lesson](./algorithm_development.md), you have created a simple
+algorithm. For this challenge, learn how to document your algorithm.
+
+- Make sure a documentation template is available in the algorithm repository. If it is
+  not, you can generate it using the `v6 algorithm` command.
+- Try out how you can run the documentation locally in your browser, and adapt it to
+  your needs.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Visualize the results of your algorithm in the UI
+
+In the [previous lesson](./algorithm_development.md), you have created a simple
+algorithm. In this challenge, modify the algorithm in the algorithm store to include
+a visualization of the results.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Expand your average algorithm
+
+In the [previous lesson](./algorithm_development.md), you have created a simple
+average algorithm. In this challenge, expand your algorithm to calculate the one or
+more of the following:
+
+- Median
```

Comment:
Haha your *mean*.

Reply:
I guess I didn't think about computating a federate median - that's not trivial I think? I just removed it now.